src/main/java/Test.java

```diff
@@ -0,0 +1,77 @@
+/*
```

Comment:
Could you include steps (in the readme) on:

1. How to build and publish this project (probably link some guide)
2. How to run this test app